### PR TITLE
treewide: fix derivation names

### PR DESCRIPTION
**Pull request by ryantm**
###### Motivation for this change

all these packages were being excluded from repology because their derivation names could not be parsed.

###### Things done

- [x] built all of them on NixOS

### CODE REVIEW

**Comment by ryantm** [comment]
@GrahamcOfBorg build altcoins.mist infamousPlugins swh-lv2 manuskript neovim-remote

**Comment by ryantm** [comment]
@GrahamcOfBorg build wxhexeditor yi exrtools gnuclad cura-lulzbot nix-tour xrandr-invert-colors

**Comment by ryantm** [comment]
@GrahamcOfBorg build uzbl coyim salut_a_toi weechat gwyddion ripser

File: pkgs/applications/editors/manuskript/default.nix [hunk 1]
```diff
@@ -1,11 +1,12 @@
 { stdenv, zlib, fetchFromGitHub, python3Packages }:
 
 python3Packages.buildPythonApplication rec {
-  name = "manuskript";
+  pname = "manuskript";
+  name = "${pname}-${version}";
```
Comment (veprbl) [inline pkgs/applications/editors/manuskript/default.nix]: Like with ```buildPythonPackage``` you don't need to define ```name``` here.

File: pkgs/applications/science/misc/simgrid/default.nix [hunk 2]
```diff
@@ -14,18 +14,17 @@ with stdenv.lib;
 
 let
   optionOnOff = option: "${if option then "on" else "off"}";
-  urlVersion = replaceStrings ["."] ["_"];
 in
 
 stdenv.mkDerivation rec {
   name = "simgrid-${version}";
-  version = "3.17";
+  version = "3.19.1";
```
Comment (FRidh) [inline pkgs/applications/science/misc/simgrid/default.nix]: version bump

**Review by FRidh — APPROVED** [review]
I checked the changes and they seem good. Note there is a version bump inside as well.

File: pkgs/applications/science/misc/simgrid/default.nix (hunk 2 above)
Comment (ryantm) [reply pkgs/applications/science/misc/simgrid/default.nix]: Yes, sorry for not mentioning that in the PR text it used to be more clear before I fixed the merge conflicts with master. I'll move this into it's own PR.

**Comment by ryantm** [comment]
I moved the simgrid version bump to its own PR. https://github.com/NixOS/nixpkgs/pull/41178

**Pull request merged**